src/uwtools/cli/run_forecast.py

```diff
@@ -60,11 +70,23 @@ def parse_args(args: List[str]) -> Namespace:
         metavar="FILE",
     )
     optional.add_argument(
-        "-m",
-        "--machine",
-        help="Path to YAML platform-definition file",
+        "--forecast-app",
+        choices={"HAFS", "MRW", "SRW"},
+        help="The app to be run",
+    )
+    optional.add_argument(
+        "--forecast-model",
+        choices={"CCPP", "CICE", "CMEPS", "FV3", "MOM6"},
+        help="The experiment to be run",
+    )
```

Comment:
Is this actually a _required_ argument? Can we do without it?

Also, I think it would be better not to add support for choices we don't have, i.e. all the models _except_ for `FV3`. We can easily add those later when we have code support for them. If we release this, users get the impression (from the `--help` switch) that these are supported.

That said, it's probably not worth doing much more work in this module, as it will mostly be OBE by the modal-CLI work anyway. Just getting the programmatic interface working, and unit tests validating it, should be enough for this ticket.

Reply:
I can depreciate the unsupported models for this ticket :)